### PR TITLE
file css modules added

### PR DESCRIPTION
**Pull request by davidalan004**

### CODE REVIEW

File: src/components/Blog.module.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+.layout {
+  background-color: white;
+  width: 1600px;
+  height: 700px;
+
+  
+}
+
+.container {
```
Comment (abrahamAlonzo) [inline src/components/Blog.module.css:36]: erase unused style

**Review by davidalan004 — COMMENTED** [review]
content unused eliminated

File: src/components/Blog.module.css (hunk 1 above)
Comment (abrahamAlonzo) [reply src/components/Blog.module.css]: fixed in 22aad5a8025ff6bc2616dffe76fc6a81c0a23963

**Review by abrahamAlonzo — APPROVED** [review]
LGTM!

**Pull request merged**